vast/vastutil/unwrap_test.go

```diff
@@ -29,11 +29,12 @@ func TestUnwrap(t *testing.T) {
 	}{
 		{"malformed1", []string{}, &xml.SyntaxError{Line: 4, Msg: "unexpected EOF"}},
```

Comment:
Suggest to add desc item for each test case, entry and trace are hard to understand what the test case purpose for.

Reply:
Got it. I will add some descriptions for them. Thanks!